scripts/jenkins/cloud/ansible/roles/heat-generator/templates/heat-template.yaml

```diff
@@ -88,15 +103,6 @@ resources:
         - {{ dns_server }}
 {%       endfor %}
 {%     endif  %}
-{%   endif  %}
-{%   if ipv6 %}
-      ip_version: 6
-{%     if network.is_conf or network.is_mgmt %}
-      ipv6_ra_mode: dhcpv6-stateful
-      ipv6_address_mode: dhcpv6-stateful
-{%     endif %}
-{%   else %}
-      ip_version: 4
```

Comment:
Is there a path in the logic we are missing by removing this lines?

Reply:
I don't see one. I think it looks ok.

Reply:
Ditto